### PR TITLE
Update Valkey startup infomation and other output infomation for Valk…

### PR DESCRIPTION
**Pull request by hwware**
Fix Valkey server startup message issue as below 

image

And update some output messages

We will rebase after pr https://github.com/valkey-io/valkey/pull/206 is merged

### CODE REVIEW

File: src/server.c [hunk 1]
```diff
@@ -1871,17 +1871,17 @@ void createSharedObjects(void) {
     shared.noscripterr = createObject(OBJ_STRING,sdsnew(
         "-NOSCRIPT No matching script. Please use EVAL.\r\n"));
     shared.loadingerr = createObject(OBJ_STRING,sdsnew(
-        "-LOADING Redis is loading the dataset in memory\r\n"));
+        "-LOADING Server is loading the dataset in memory\r\n"));
     shared.slowevalerr = createObject(OBJ_STRING,sdsnew(
-        "-BUSY Redis is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
+        "-BUSY Server is busy running a script. You can only call SCRIPT KILL or SHUTDOWN NOSAVE.\r\n"));
```
Comment (madolson) [inline src/server.c:1876]: At AWS we have this hardcoded. there are other places this stuff seems hardcoded: https://github.com/netty/netty/blob/fee3c39d0835ec18d7aace8776db5c5952ce8272/codec-redis/src/main/java/io/netty/handler/codec/redis/FixedRedisMessagePool.java#L44

I think this need to be more thoughtfully considered for compatibility. I'm leaning towards we just accept this as a compatibility problem forever.

**Review by madolson — COMMENTED** [review]
I think we should decouple the error changes from the other messages here, which I think are easier to make a decision about.

**Comment by hwware** [comment]
> I think we should decouple the error changes from the other messages here, which I think are easier to make a decision about.

After I rebase https://github.com/valkey-io/valkey/pull/206, I will make the changes only for Valkey server startup issue

File: src/server.c (hunk 1 above)
Comment (zuiderkwast) [reply src/server.c:1876]: Agree, don't change error replies. We'll not do that in the compat release.

Besides, I have a PR for those specifically: #206

**Comment by zuiderkwast** [comment]
I think we decided not to change any logging in the compatibility release. If we follow that strictly, that would mean we can't even change this one, which is like a title:

    "oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo"

Shall we make a few exceptions and change this line and a few others in the compat release?

Logging isn't as high risk as error messages though, is it?

File: src/server.c [hunk 2]
```diff
@@ -4526,7 +4526,7 @@ int finishShutdown(void) {
 
 
     serverLog(LL_WARNING,"%s is now ready to exit, bye bye...",
-        server.sentinel_mode ? "Sentinel" : "Redis");
+        server.sentinel_mode ? "Sentinel" : "Server");
```
Comment (enjoy-binbin) [inline src/server.c:4529]: somehow in here, i prefer Valkey?

Comment (zuiderkwast) [reply src/server.c:4529]: Agree with @enjoy-binbin. It's like a greeting. Similar to the logo and this one;

    oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo

We can replace that "starting" message with Valkey and different graphics instead of oO0Oo...

File: src/server.c [hunk 3]
```diff
@@ -5303,23 +5303,23 @@ void commandGetKeysCommand(client *c) {
 void commandHelpCommand(client *c) {
     const char *help[] = {
 "(no subcommand)",
-"    Return details about all Redis commands.",
+"    Return details about all Valkey commands.",
```
Comment (zuiderkwast) [inline src/server.c:5306]: Let's do help commands in a separate PR too. I did this without seeing your change: #216

**Comment by zuiderkwast** [comment]
This one is partially doing #207.

I believe we decided to skip this for 7.2.4 but I think we should reconsider it. We should at least not printing "Redis is starting" and the Redis logo. If we change those, we can as well change the other occurrences of Redis in the logging.

I'm going to mark this as a major decision. It is about whether to include logging changes in 7.2.4 or if we will postpone it to 8.0.0.

**Review by zuiderkwast — COMMENTED** [review]
This PR is a mix of logging, error replies and HELP output. It's better that we do this in separate PRs. There are also issues for all of these. See #25.

Some of this is already done in other PRs and merged.

@hwware Do you want to re-purpose this PR to do only the logging changes?

**Review by zuiderkwast — CHANGES_REQUESTED** [review]
Don't merge. See #207 (and other issues).

**Comment by hwware** [comment]
Since startup information has been updated in other PR, i prefer to close this PR,
